iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -157,8 +160,15 @@ public bool CertificateRevocationCheck
 
         /// <summary>
         /// The time to wait for receiving an acknowledgment for a CONNECT packet.
-        /// The default is 10 seconds.
+        /// The default is 60 seconds.
         /// </summary>
+        /// <remarks>
+        /// In the event that IoT Hub receives burst traffic, it will implement traffic shaping in order to process the incoming requests.
+        /// In such cases, during client connection the CONNECT requests can have a delay in being acknowledged and processed by IoT Hub.
+        /// The <see cref="ConnectArrivalTimeout"/> governs the duration the client will wait for a CONNACK packet before disconnecting and reopening the connection.
+        /// To know more about IoT Hub's throttling limits and traffic shaping feature, see
+        /// <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling#operation-throttles"/>.
```

Comment:
remove en-us from link

Reply:
Thanks for highlighting this Basel, I seem to miss a couple in each PR! :woman_facepalming:

Reply:
I only started paying attention to this kind of feedback recently when someone made a comment on my PR :).